pages/tools/cli.md

```diff
@@ -4,7 +4,7 @@ This tool allows you to interact with the resin.io api from the comfort of your
 
 Please make sure your system meets the requirements as specified in the [README](https://github.com/resin-io/resin-cli).
 
-To get started, download the CLI from npm:
+To get started download the CLI from npm.
```

Comment:
might be worth adding @pimterry as a reviewer for this section on CLI docs?

Reply:
:+1:  When reviewing this, I glossed over most of the CLI and SDK docs ;-)

Reply:
These get pulled in from the separate CLI repo, so any changes we need to make should be requested there. I'll be making a PR soon to make sure Docker is capitalized everywhere in that repo